app-modules/engagement/src/Filament/Actions/BulkEngagementAction.php

```diff
@@ -89,7 +90,25 @@ public static function make(string $context)
                             ->required()
                             ->placeholder(__('Subject'))
                             ->hidden(fn (Get $get): bool => $get('channel') === NotificationChannel::Sms->value)
-                            ->columnSpanFull(),
+                            ->columnSpanFull()
+                            ->visible(! RefactorEngagementCampaignSubjectToJsonb::active()),
+                        TiptapEditor::make('subject')
+                            ->label('Subject')
+                            ->mergeTags([
+                                'student first name',
+                                'student last name',
+                                'student full name',
+                                'student email',
+                                'student preferred name',
```

Comment:
Due to a recent change in one of Dan's ticket, this should now be `recipient` instead of `student`.